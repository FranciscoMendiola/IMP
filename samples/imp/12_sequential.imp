x := 1;
y := 2;
z := x + y;
resultado := z * 10
