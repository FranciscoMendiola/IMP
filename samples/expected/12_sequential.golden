Token {tokenKind = Id, lexeme = "x", position = 0}
Token {tokenKind = Assign, lexeme = ":=", position = 2}
Token {tokenKind = IntLit, lexeme = "1", position = 5}
Token {tokenKind = Semi, lexeme = ";", position = 6}
Token {tokenKind = Id, lexeme = "y", position = 8}
Token {tokenKind = Assign, lexeme = ":=", position = 10}
Token {tokenKind = IntLit, lexeme = "2", position = 13}
Token {tokenKind = Semi, lexeme = ";", position = 14}
Token {tokenKind = Id, lexeme = "z", position = 16}
Token {tokenKind = Assign, lexeme = ":=", position = 18}
Token {tokenKind = Id, lexeme = "x", position = 21}
Token {tokenKind = Plus, lexeme = "+", position = 23}
Token {tokenKind = Id, lexeme = "y", position = 25}
Token {tokenKind = Semi, lexeme = ";", position = 26}
Token {tokenKind = Id, lexeme = "resultado", position = 28}
Token {tokenKind = Assign, lexeme = ":=", position = 38}
Token {tokenKind = Id, lexeme = "z", position = 41}
Token {tokenKind = Times, lexeme = "*", position = 43}
Token {tokenKind = IntLit, lexeme = "10", position = 45}
